.eslintrc.ts

```diff
@@ -0,0 +1,55 @@
+module.exports = {
```

Comment:
Can't we use TypeScript exports instead?

Reply:
looks like it is more natural to use json